### DISCUSSION TITLE
syncbatchnorm0_gamma' has not been initialized yet because initialization was deferred

### DISCUSSION THREAD

**Discussion by ngunauj** [apache/incubator-mxnet#15665] [Q&A]
The part of forward..
```
    def forward(self, x):
        x = self.conv1(x)
        x = self.bn1(x)
        x = self.relu(x)
        x = self.conv2(x)
        x = self.bn2(x)
        x = self.relu(x)
```
The part of define..
```
        self.conv1 = nn.Conv2D(
            channels=64,
            kernel_size=3,
            strides=2,
            padding=1,
            in_channels=3,
            use_bias=False)
        self.bn1 = BatchNorm2d(in_channels=64, momentum=BN_MOMENTUM)
        self.conv2 = nn.Conv2D(
            channels=64,
            kernel_size=3,
            strides=2,
            padding=1,
            in_channels=64,
            use_bias=False)
        self.bn2 = BatchNorm2d(in_channels=64, momentum=BN_MOMENTUM)
        self.relu = nn.Activation('relu')
```

the BatchNorm2d is
```
from mxnet.gluon.contrib.nn import SyncBatchNorm
BatchNorm2d = functools.partial(SyncBatchNorm, num_devices=4)
```
when run the code, the error is `syncbatchnorm0_gamma' has not been initialized yet because initialization was deferred`.
when i change the SyncBatchNorm to BatchNorm is ok....

i use `net.initialize(ctx=[mx.cpu(0)])` to init params
